### ISSUE TITLE
Filter multiple matplotlib plot using qgird

### ISSUE THREAD

**Issue by RuilinOuyang** [quantopian/qgrid#256]
I found there's a way to create an interactive plot by using qgrid. But all the examples in binder are only has a plot. Is it possible to have multiple plot that can interact with qgrid filter?

**Comment by zxzhaixiang** [comment]
Yes. Simply update those plots together in the callback function
